enterprise/server/test/integration/remote_bazel/remote_bazel_test.go

```diff
@@ -140,7 +140,8 @@ func runRemoteBazelInSeparateProcess(t *testing.T, workDir string, serverAddress
 			"--runner_exec_properties=container-image=",
 		},
 		args...)...)
-	err := cmd.Run()
+	b, err := cmd.CombinedOutput()
+	fmt.Println(string(b))
```

Comment:
nit: you might want to use `t.Log` here? I think we typically don't write things to stdout (e.g. the `log` package writes everything to stderr)